angel_murillo.md

```diff
@@ -0,0 +1 @@
+A front end developer creates websites and applications using web languages like HTML, CSS, and JavaScript that allow users to access and interact with the site or app.
```

Comment:
I also agree that Front-End Development is basically HTML, CSS, and JavaScript! Great explanation